R/hiper_glm.R

```diff
@@ -12,22 +28,32 @@
 #' @return A list including the information of the fitted model
 #'
 #' @export
-hiper_glm <- function(design, outcome, model = "linear") {
+hiper_glm <- function(design, outcome, model = "linear", option = NULL) {
   supported_model <- c("linear", "logit")
```

Comment:
```suggestion
  supported_model <- c("linear")
```

You can also try using something like `match.arg()` here